src/redux/sagas/ws/on_all_tickers_subscribe.js

```diff
@@ -0,0 +1,25 @@
+import { put, delay } from 'redux-saga/effects'
+import Debug from 'debug'
+import axios from 'axios'
+import WSActions from '../../actions/ws'
+
+const debug = Debug('hfui:rx:s:ws-hfui:on-all-tickers-subscribe')
+
+const UPDATE_RATES_INTERVAL_MS = 10 * 1000 // 10s
+const URL = 'http://localhost:45001/v2/tickers?symbols=ALL'
+
+export default function* () {
+  while (true) {
+    let tickersData
+
+    try {
+      const response = yield axios.get(URL)
+      tickersData = response.data
+    } catch (err) {
+      debug('failed to fetch tickers rates: %s', err.message)
+      return
+    }
+    yield put(WSActions.setTickersData(tickersData))
+    yield delay(UPDATE_RATES_INTERVAL_MS)
```

Comment:
Do we really need a return in catch statement? I think it would be better to refactor the code into something like that:
```suggestion
    try {
      const { data } = yield axios.get(URL)
      yield put(WSActions.setTickersData(data))
    } catch (err) {
      debug('failed to fetch tickers rates: %s', err.message)
    }
    yield delay(UPDATE_RATES_INTERVAL_MS)
```